tests/everflow/everflow_test_utilities.py

```diff
@@ -725,6 +725,8 @@ def get_expected_mirror_packet(mirror_session, setup, duthost, mirror_packet, ch
         expected_packet.set_do_not_care_scapy(packet.IP, "len")
         expected_packet.set_do_not_care_scapy(packet.IP, "flags")
         expected_packet.set_do_not_care_scapy(packet.IP, "chksum")
+        if duthost.facts["asic_type"] == 'marvell':
```

Comment:
might need consider marvell-armhf

Reply:
Hi @Blueve

I have checked the ansible facts gathered by the code. facts asic_type is marvell and asic_subtype is marvell-armf. These changes are common for all marvell devices. I think we should use asic_type=marvell here

Please find details given below:-

ansible_facts": {"discovered_interpreter_python": "/usr/bin/python", "dut_basic_facts": {"asic_subtype": "marvell-armhf", "asic_type": "marvell", "branch": "202205", "build_date": "Thu Nov 24 12:44:20 UTC 2022", "build_number": 179361, "build_version": "202205.179361-7c1015b45

Thanks
Rajneesh